src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/recommendations/RecommendationsFragment.kt

```diff
@@ -36,10 +37,10 @@ class RecommendationsFragment : Fragment() {
 
         var listView = binding.recommendedListView
         activity?.let {
-            val adapter = ArrayAdapter<String>(
-                it,
-                R.layout.simple_list_item_1,
-                recommendedViewModel.mangasNames
+            val adapter = MangaListAdapter(
+                requireActivity(),
```

Comment:
if you use requireActivity please do not use activity?.let

Reply:
there will be error?

Reply:
No, but activity?.let and requireActivity are two ways to do same things